src/GlobalFunctions.php

```diff
@@ -214,20 +214,19 @@ function smwfGetLinker() {
  * @codeCoverageIgnore
  */
 function enableSemantics( $namespace = null, $complete = false ) {
-	global $smwgNamespace;
 
 	// Apparently this is required (1.28+) as the earliest possible execution
 	// point in order for settings that refer to the SMW_NS_PROPERTY namespace
 	// to be available in LocalSettings
 	NamespaceManager::initCustomNamespace( $GLOBALS );
 
-	if ( !$complete && ( $smwgNamespace !== '' ) ) {
+	if ( !$complete && ( $GLOBALS['smwgNamespace'] !== '' ) ) {
```

Comment:
@mwjames This change causes:
```
PHP Notice:  Undefined index: smwgNamespace in /../w/extensions/SemanticMediaWiki/src/GlobalFunctions.php on line 223
Notice: Undefined index: smwgNamespace in /../w/extensions/SemanticMediaWiki/src/GlobalFunctions.php on line 223
```

Reply:
Interesting, just a sec.

Reply:
Quick question, what happens if we change it back to?

```
function enableSemantics( $namespace = null, $complete = false ) {
	global $smwgNamespace;

	// Apparently this is required (1.28+) as the earliest possible execution
	// point in order for settings that refer to the SMW_NS_PROPERTY namespace
	// to be available in LocalSettings
	NamespaceManager::initCustomNamespace( $GLOBALS );

	if ( !$complete && ( $smwgNamespace !== '' ) ) {
		// The dot tells that the domain is not complete. It will be completed
		// in the Export since we do not want to create a title object here when
		// it is not needed in many cases.
		$smwgNamespace = '.' . $namespace;
	} else {
		$smwgNamespace = $namespace;
	}

	$GLOBALS['smwgSemanticsEnabled'] = true;

	return true;
}
```

Reply:
Silence. At least cron does not longer fire away these notices. The wiki seems to be fine, too.

Reply:
>Silence. At least cron does no longer fire away these notices. The wiki seems to be fine, too.

In that case #2894.

Reply:
I just updated the wiki manually to assure that continuous automatic updating does not stop. Thanks for fix!